src/main/java/io/jenkins/plugins/analysis/warnings/axivion/RemoteAxivionDashboard.java

```diff
@@ -65,12 +69,20 @@ public JSONObject getIssues(final AxIssueKind kind) {
         catch (IOException e) {
             throw new ParsingException(e, "Cannot retrieve information from dashboard");
         }
-        catch (JSONException e) {
-            throw new ParsingException(e, "Invalid JSON response from dashboard");
-        }
     }
 
-    private JSONObject convertToJson(final HttpResponse response) throws IOException {
-        return JSONObject.fromObject(EntityUtils.toString(response.getEntity()));
```

Comment:
This was the problematic line. Internally it created thousands of uncached Regex and was the bottleneck.